aws-xray-auto-instrumentation-agent-bootstrap/src/main/java/com/amazonaws/xray/agent/XRayInstrumentationAgent.java

```diff
@@ -93,7 +93,7 @@ private static boolean initializeRuntimeAgent(String agentArgs, ClassLoader clas
     private static AgentRuntimeLoaderInterface getAgentRuntimeLoader(ClassLoader classLoader) throws ClassNotFoundException, IllegalAccessException, InvocationTargetException, InstantiationException, NoSuchMethodException {
         // Iterate through its parents until we find it. If we reach the bootstrap, then we know it doesn't exist.
 
-        Class<?> agentRuntimeLoaderClass = null;
+        Class<?> agentRuntimeLoaderClass;
```

Comment:
Optional would just move this to the initialization line

Reply:
Good call, done.